### PR TITLE
Add PixelShuffle for load lua

### PR DESCRIPTION
**Pull request by akshayMpatel**
fellow contributors @tsachiblau @KatiePodiatchev

### CODE REVIEW

File: torch/legacy/nn/PixelShuffle.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        else:
+            tmp = self._outShape.type(torch.IntTensor)
+            tmp = torch.Size((tmp[0], tmp[1], tmp[2]))
+        self.output = self._shuffleOut.view(tmp)
+
+        print("the r factor is : ", self.upscaleFactor)
+        print("the input is: ", input.size())
+        print("input type: ", input.type())
+        print("the output is: ", self.output.size())
+        print("output type: ", self.output.type())
```
Comment (apaszke) [inline torch/legacy/nn/PixelShuffle.py]: You need to remove prints used for debugging

File: torch/utils/serialization/read_lua_file.py [hunk 2]
```diff
@@ -366,6 +366,12 @@ def storage_to_size(obj, *attrs):
         setattr(obj, attr, torch.Size(value))
 
 
+@registry_addon
+def storage_to_FP(obj, attr):
+    value = getattr(obj, attr)
+    setattr(obj, attr, value)
```
Comment (apaszke) [inline torch/utils/serialization/read_lua_file.py]: sorry, but what is this function supposed to do? looks like a no-op to me

File: torch/legacy/nn/PixelShuffle.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
+import torch
+from .Module import Module
+from torch.legacy.nn.View import View
+import numpy
```
Comment (apaszke) [inline torch/legacy/nn/PixelShuffle.py]: Please remove this import

**Review by apaszke — CHANGES_REQUESTED** [review]

File: torch/legacy/nn/PixelShuffle.py (hunk 3 above)
Comment (ak096) [reply torch/legacy/nn/PixelShuffle.py]: Done

File: torch/legacy/nn/PixelShuffle.py (hunk 1 above)
Comment (ak096) [reply torch/legacy/nn/PixelShuffle.py]: Done

File: torch/utils/serialization/read_lua_file.py (hunk 2 above)
Comment (ak096) [reply torch/utils/serialization/read_lua_file.py]: sorry for that, yes just wanted to add PixelShuffle to the reader registry, now we just used the ensure_attr function to do this

**Review by akshayMpatel — COMMENTED** [review]
@apaszke - your change requests and question have been addressed - please review

**Comment by ak096** [comment]
@apaszke - your change requests and question have been addressed - please re

**Comment by soumith** [comment]
@ak096 please test your changes locally, all the builds are failing.

**Comment by ak096** [comment]
noticed the other Modules in pytorch/torch/legacy/nn/ have imports in them, which contradicts the second change request from @apaszke 
will include them again

**Comment by ezyang** [comment]
Still failing.

**Comment by fmassa** [comment]
We have removed support for `legacy` in PyTorch 1.0.
I'm closing this PR because of that.

Thanks anyway for the PR!

**Pull request closed without merging**